test/formatters.test.ts

```diff
@@ -0,0 +1,34 @@
+import { it, describe, expect } from 'vitest';
```

Comment:
In monorepo code tests live alongside the code and not in a separate folder. Is this a decision to change that here or an oversight?